transport/src/iosMain/kotlin/com/genesys/cloud/messenger/transport/util/EncryptedVault.ios.kt

```diff
@@ -0,0 +1,38 @@
+package com.genesys.cloud.messenger.transport.util
+
+import com.genesys.cloud.messenger.transport.core.InternalVault
+
+/**
+ * iOS implementation of EncryptedVault that wraps the iOS default vault (Keychain).
+ */
+actual class EncryptedVault actual constructor(keys: Keys) :
```

Comment:
Would the following be simpler and still working? 

```suggestion
actual class EncryptedVault actual constructor(keys: Keys) : InternalVault(keys)
```

Reply:
umm not sure I understand , the Encrypted the actual class of the expect class 
![image](https://github.com/user-attachments/assets/49050cd3-498a-459d-9a8a-6e85b2428934)

so it cannot be inherited from InternalVault